packages/gatsby/src/commands/build.ts

```diff
@@ -35,16 +35,16 @@ if (process.env.GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES) {
   cachedWebpackCompilationHash = webpackCompilationHash
 }
 
-type BuildArgs = {
-  directory: string,
-  sitePackageJson: object,
-  prefixPaths: boolean,
-  noUglify: boolean,
-  profile: boolean,
-  openTracingConfigFile: string,
+interface IBuildArgs extends IProgram {
+  directory: string
+  sitePackageJson: object
```

Comment:
There are typings for `PackageJson` in index.d.ts, which would be better to use here.